### PR TITLE
*: Upgrade to Thanos v0.30.1

### PR DESCRIPTION
**Pull request by douglascamata**
Signed-off-by: Douglas Camata <159076+douglascamata@users.noreply.github.com>

* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Upgrades Thanos from v0.29 to v0.30.1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#279](https://github.com/thanos-io/kube-thanos/pull/279) Change `hashringConfigmapName` to `hashringConfigMapName` in Receive configuration.
 - [#284](https://github.com/thanos-io/kube-thanos/pull/284) Change Receive `PodDisruptionBudget` api version to `policy/v1`
+- [#293](https://github.com/thanos-io/kube-thanos/pull/293) Upgrade to Thanos v0.30.1
```
Comment (yeya24) [inline CHANGELOG.md:22]: Let's upgrade to v0.30.2?

Comment (douglascamata) [reply CHANGELOG.md:22]: Suuuure! Going for the v0.30.2 ugprade. 💪

Comment (douglascamata) [reply CHANGELOG.md:22]: @yeya24 done in https://github.com/thanos-io/kube-thanos/pull/293/commits/6bf393641fee0d0dc76bddfce5727a7089694094

**Review by saswatamcode — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (saswatamcode) [reply CHANGELOG.md:22]: Hmm seems like the changelog needed to updated. No worries, raised a PR: https://github.com/thanos-io/kube-thanos/pull/294

Comment (douglascamata) [reply CHANGELOG.md:22]: Thx for the follow up with the fix, @saswatamcode 🙇

**Pull request merged**